### PR TITLE
Footer Created

### LINKED ISSUE #3: Add Some Footer Content
Please add some footer content in homepage
Footer content
     1. Facebook
      2. Twitter
      3. About

Capture

@rajat2502 Please review the issue

### PR DESCRIPTION
**Pull request by ankushbhardwxj**
Resolves #3 
Designed a footer content with social handles & also with list of contributors.
- Added instagram, facebook, youtube and github. Only github icon redirects to github page.
- Added an `anchor` element which redirects to list of contributors on github
note : The icons are animated.
This is how it looks
Capture

### CODE REVIEW

**Comment by ankushbhardwxj** [comment]
@rajat2502 please review and let me know for any changes 👍

**Comment by rajat2502** [comment]
Remove all links except the GitHub contributors list link.
Also, add the link to the main repo.
Style the footer, Keep it dark grey maybe

**Comment by ankushbhardwxj** [comment]
@rajat2502 link to main repo is done by clicking the github icon.
Also this colour matches the theme quite nicely. Dark grey might be odd , since the entire page has light colours.

**Comment by ankushbhardwxj** [comment]
@rajat2502 I think it looks really good now 👍 
Revenous-GoogleChrome2019-12-111

File: src/components/Footer/Footer.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import SocialFollow from "./SocialFollow";
+import "./style.css";
```
Comment (rajat2502) [inline src/components/Footer/Footer.js:3]: No need of importing style.css again as it is already imported in the App component

File: src/components/Footer/SocialFollow.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  faGratipay
+} from "@fortawesome/free-brands-svg-icons";
+
+export default function SocialFollow() {
+  return (
+    <div className="social-container">
+      <h3>See source code at<a href="https://github.com/rajat2502/Ravenous" className="github social">
+        <FontAwesomeIcon icon={faGithub} size="2x" />
+      </a></h3>
+      <br></br>
```
Comment (rajat2502) [inline src/components/Footer/SocialFollow.js:14]: write it as <br /> instead of a closing tag

**Review by rajat2502 — CHANGES_REQUESTED** [review]

File: src/components/Footer/SocialFollow.js (hunk 2 above)
Comment (rajat2502) [reply src/components/Footer/SocialFollow.js:14]: Also, remove the love icon from the footer

**Comment by rajat2502** [comment]
I don't see any updates.
Closing it.

**Comment by ankushbhardwxj** [comment]
@rajat2502 sorry for delay. Don't you think this footer is better compared to the one that has been implemented? If you'd like I can make the changes now, Let me know about this.

**Pull request closed without merging**